### PR TITLE
Update login-rate-limit.md

### PR DESCRIPTION
**Pull request by animir**
I put new example on top as `express-brute` package is actually doesn't do this job nicely. 
It delays requests not block them.

### CODE REVIEW

**Comment by js-kyle** [comment]
Awesome! I will review this shortly

File: README.md [hunk 1]
```diff
@@ -801,7 +801,7 @@ All statements above will return false if used with `===`
 
 <a href="https://www.owasp.org/index.php/Top_10-2017_A2-Broken_Authentication" target="_blank"><img src="https://img.shields.io/badge/%E2%9C%94%20OWASP%20Threats%20-%20A9:Broken%20Authentication%20-green.svg" alt=""/></a>
 
-**TL;DR:** A brute force protection middleware such as [express-brute](https://www.npmjs.com/package/express-brute) should be used inside an express application to prevent brute force/dictionary attacks on sensitive routes such as /admin or /login based on request properties such as the user name, or other identifiers such as body parameters
+**TL;DR:** A brute force protection such as [rate-limiter-flexible](https://www.npmjs.com/package/rate-limiter-flexible) package or middleware such as [express-brute](https://www.npmjs.com/package/express-brute) for express application should be used to prevent brute force/dictionary attacks on sensitive routes such as /admin or /login based on request properties such as the user name, or other identifiers such as body parameters
```
Comment (BrunoScheufler) [inline README.md]:
```suggestion
**TL;DR:** A brute force protection such as [rate-limiter-flexible](https://www.npmjs.com/package/rate-limiter-flexible) package or middleware such as [express-brute](https://www.npmjs.com/package/express-brute) for express applications should be used to prevent brute force/dictionary attacks on sensitive routes such as /admin or /login based on request properties such as the user name, or other identifiers such as body parameters
```
`application` -> `applications`

**Comment by i0natan** [comment]
@animir See response in #308

**Comment by animir** [comment]
Cancel this PR, as I'm going to rewrite this example
